yt/docs/ru/toc.yaml

```diff
@@ -336,6 +336,9 @@ items:
       - name: Импорт внешних данных
         when: audience == "public"
         href: user-guide/import.md
+      - name: Query tracker
+        when: audience == "internal"
```

Comment:
Should we make it public since the beginning?

Reply:
All imports in this file are intended for public use.

Also, this documentation has more sense in the public domain, so let's make it public too.